protocol/messenger_mention.go

```diff
@@ -30,6 +30,8 @@ const (
 	charCodeBlock  = "`"
 
 	intUnknown = -1
+
+	suggestionsLimit = 15
```

Comment:
Does it really have to be 15? On Discord I think the max number of mentions is 10.